doc/manual/src/command-ref/nix-copy-closure.md

```diff
@@ -14,62 +14,57 @@
 
 # Description
 
-`nix-copy-closure` gives you an easy and efficient way to exchange
-software between machines.  Given one or more Nix store _paths_ on the
-local machine, `nix-copy-closure` computes the closure of those paths
-(i.e. all their dependencies in the Nix store), and copies all paths
-in the closure to the remote machine via the `ssh` (Secure Shell)
-command.  With the `--from` option, the direction is reversed: the
-closure of _paths_ on a remote machine is copied to the Nix store on
-the local machine.
-
-This command is efficient because it only sends the store paths
-that are missing on the target machine.
-
-Since `nix-copy-closure` calls `ssh`, you may be asked to type in the
-appropriate password or passphrase.  In fact, you may be asked _twice_
-because `nix-copy-closure` currently connects twice to the remote
-machine, first to get the set of paths missing on the target machine,
-and second to send the dump of those paths.  When using public key
-authentication, you can avoid typing the passphrase with `ssh-agent`.
+`nix-copy-closure` copies [store objects](@docroot@/glossary.md#gloss-store-object) along with all their dependencies to or from another machine via the SSH protocol.
+It doesn’t copy store objects that are already present on the target machine.
+
+Given one or more _paths_ accesible from the client, `nix-copy-closure` computes the [closure](@docroot@/glossary.md#gloss-closure) of those paths (i.e. all their dependencies in the Nix store), and copies all store objects in the closure to the remote machine via SSH.
+With the `--from` option, the direction is reversed:
```

Comment:
:anger: why do we start with wrong statements just to correct them afterwards?

Reply:
Classic symptom of conservative incremental changes. But rewriting all of documentation doesn't get you merges...